nubi/src/components/buttons/about/button.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styles from './styles.module.scss';
+
+function AboutButton() {
+  return (
+   <button className={styles.button}><p1 className={styles.text}>Ver más</p1></button>
```

Comment:
Recuerda siempre usar tags apropiados para el contenido. `p1` no es un tag de HTML válido. Acá podrías dejar sólo el contenido del botón sin necesidad de hacer un wrapping adicional.
```suggestion
   <button className={styles.button}>Ver más</button>
```